src/formatting.rs

```diff
@@ -705,7 +770,24 @@ fn make_parse_sess(source_map: Rc<SourceMap>, config: &Config) -> ParseSess {
         } else {
             ColorConfig::Never
         };
-        Handler::with_tty_emitter(color_cfg, true, None, Some(source_map.clone()))
+
+        let emitter_writer =
+            EmitterWriter::stderr(color_cfg, Some(source_map.clone()), false, false, None);
```

Comment:
this is the emitter that `Handler::with_tty_emitter` creates internally, so creating it here and including it in the custom `Emitter` info so that parser errors that occur in non-ignored files are still displayed as usual